File: src/server/MovieTicket.Domain/Entities/Seat.cs
using MovieTicket.Core.Domain;

namespace MovieTicket.Domain.Entities;

public class Seat : EntityBase
{
    public int RowNumber { get; init; }
    public int ColNumber { get; init; }
    
    public Guid RoomId { get; init; }
    public virtual Room Room { get; init; }
    
    
    public virtual ICollection<SeatReservation> SeatReservations { get; init; }
}

File: src/client/mobile/app/src/main/java/com/superman/movieticket/domain/entities/Seat.kt
package com.superman.movieticket.domain.entities

import com.superman.movieticket.core.entity.BaseEntity
import java.io.Serializable

data class Seat (
    val id: String,
    val rowNumber: String,
    val colNumber: Number,
    val isPlaced: Boolean
):Serializable